geometry/include/pcl/geometry/mesh_indices.h

```diff
@@ -38,314 +38,54 @@
  *
  */
 
-// NOTE: This file has been created with
-// 'pcl_src/geometry/include/pcl/geometry/mesh_indices.py'
-
 #pragma once
 
 #include <pcl/geometry/boost.h>
 
 #include <iostream>
 
 ////////////////////////////////////////////////////////////////////////////////
-// VertexIndex
+// MeshIndex
 ////////////////////////////////////////////////////////////////////////////////
 
 namespace pcl {
-namespace geometry {
+namespace detail {
 /** \brief Index used to access elements in the half-edge mesh. It is basically just a
  * wrapper around an integer with a few added methods.
  * \author Martin Saelzle
- * \ingroup geometry
+ * \ingroup detail
```

Comment:
You can just repeat the doc-string near the declarations (L189)

Reply:
Regarding this modification, what should I do? Do you want me to put the `\ingroup detail` for each class method?

Reply:
in group geometry, just like usual. Basically, no change in the generated documentation